### ISSUE TITLE
UmbrelOS install on Ubuntu Server Fails

### ISSUE THREAD

**Issue by ripkova** [getumbrel/umbrel#1779]
`curl -L https://umbrel.sh | bash -s -- --no-install-docker --no-install-compose`

Here is the error:
```
======================================
============= STARTING ===============
============== UMBREL ================
======================================

Setting environment variables...

Starting karen...

Starting status monitors...
Starting memory monitor...

Starting backup monitor...

Starting decoy backup trigger...

Starting umbreld...


Starting Docker services...

Traceback (most recent call last):
  File "/usr/local/bin/docker-compose", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.10/dist-packages/compose/cli/main.py", line 81, in main
    command_func()
  File "/usr/local/lib/python3.10/dist-packages/compose/cli/main.py", line 200, in perform_command
    project = project_from_options('.', options)
  File "/usr/local/lib/python3.10/dist-packages/compose/cli/command.py", line 60, in project_from_options
    return get_project(
  File "/usr/local/lib/python3.10/dist-packages/compose/cli/command.py", line 152, in get_project
    client = get_client(
  File "/usr/local/lib/python3.10/dist-packages/compose/cli/docker_client.py", line 41, in get_client
    client = docker_client(
  File "/usr/local/lib/python3.10/dist-packages/compose/cli/docker_client.py", line 124, in docker_client
    kwargs = kwargs_from_env(environment=environment, ssl_version=tls_version)
TypeError: kwargs_from_env() got an unexpected keyword argument 'ssl_version'
Failed to start containers
Skipping status update when not on Umbrel OS
```

**Comment by highghlow** [comment]
This might be related to #1778 

This error originates from the docker-compose python package which hasn't been updated in 3 years. Debian packages a newer version of docker-compose. I recommend following the recommendations in #1778, I have tested them on Debian 12 (Ubuntu is based on Debian)

**Comment by highghlow** [comment]
In your case you skip installing docker-compose, so your should remove the python package (`pip uninstall docker-compose`) and install the Debian one (`sudo apt install docker-compose`)

**Comment by ripkova** [comment]
Same issue still occurs when using the apt version.

**Comment by calebdame-bp** [comment]
same issue here, also after using the fix @ripkova proposed.  I am on Ubuntu 22.04

**Comment by ripkova** [comment]
Same aswell, I'm on ubuntu server 22.04 lts.

**Comment by mayankchhabra** [comment]
Sorry for the issue folks. It's now been fixed by #1780.
